### PR TITLE
541/gg avax redemption

### PR DESCRIPTION
**Pull request by jacktrusler**
- Made the liquid staking page inform the user when the liquidity pool is out of avax and cannot swap from ggAvax --> Avax
- Link to balancer page when they're out of funds
- Moved documentation sidebar link to bottom of page
- Added hook to fetch ggAvax balance

### CODE REVIEW

**Review by chand1012 — APPROVED** [review]

File: src/hooks/ggAvaxBalance.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { useBalance } from 'wagmi'
+
+//Address is the tokenGgAvax contract
+//Token is the wAVAX contract
+function useGgAvaxBalance() {
+  const balance = useBalance({
+    address: '0xA25EaF2906FA1a3a13EdAc9B9657108Af7B703e3',
```
Comment (emersoncloud) [inline src/hooks/ggAvaxBalance.ts:7]: To get this address you can use the hook we made ```useTokenggAVAXContract```

File: src/hooks/ggAvaxBalance.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    token: '0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7',
```
Comment (emersoncloud) [inline src/hooks/ggAvaxBalance.ts:8]: And I'd maybe just name this one. We don't store the WAVAX address but it'd be nice to know at a glance that's what this is

**Review by emersoncloud — APPROVED** [review]
Looks good, but I think we shouldn't hardcode the ggAVAX address

File: src/hooks/ggAvaxBalance.ts (hunk 2 above)
Comment (jacktrusler) [reply src/hooks/ggAvaxBalance.ts]: I put this into contractAddresses.ts

**Pull request merged**